VideoLocker/res/layout/row_course_list.xml

```diff
@@ -220,16 +175,35 @@
         </LinearLayout>
     </RelativeLayout>
 
+    <org.edx.mobile.view.custom.ETextView
+        android:id="@+id/course_certified_banner"
+        style="@style/bold_white_text"
+        android:layout_width="wrap_content"
```

Comment:
We can get the same effect by changing the left/right margins to padding, and defining a `minWidth` attribute to have the original width + padding (so left/right padding 40 dips, width `wrap_content` and min width 180 dips).